Lesson26/Kubra/script.js

```diff
@@ -28,17 +21,38 @@ function fetchPosts() {
 				paragraph.innerText = post.body;
 				listItem.appendChild(paragraph);
 
+				const editButton = document.createElement('a');
+				editButton.classList.add('button button--danger button--edit');
```

Comment:
Minor issue here, can be fixed like this:
```suggestion
								editButton.classList.add('button', 'button--danger', 'button--edit');
```
Different class names have to be passed as array of strings to be processed correctly.